.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+npx lint-staged
```

Comment:
Not sure it makes a difference but should this say `yarn run lint-staged` since we're using yarn on this repo?

Reply:
I think you might be able to just to `lint-staged` without `npx` or `yarn run` in front of it.

Reply:
Forget what I said, looks like Yarn + Husky have Windows compat issues. https://typicode.github.io/husky/#/?id=yarn-on-windows